aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js

```diff
@@ -79,7 +79,9 @@ const PageContent = () => {
       <Grid gap={parentGrid.gap[size]} columns={parentGrid.columns[size]}>
         {/* Content Block 1 is top priority content. At narrow breakpoints, 
         place as first content element. Otherwise, place in second column. */}
-        {(size === 'small' || size === 'xsmall') && <ContentBlock title="1" />}
+        {(['xsmall', 'small'].includes(size) || size === 'xsmall') && (
```

Comment:
I think the size === 'xsmall' is redundant now.

```suggestion
        {['xsmall', 'small'].includes(size) && (
```